src/network/connection.cpp

```diff
@@ -2089,12 +1981,57 @@ void * ConnectionReceiveThread::Thread()
 	return NULL;
 }
 
+bool ConnectionReceiveThread::wait(int timeout_us, fd_set &readset) {
+	// Initialize the set
+	FD_ZERO(&readset);
+	int max_fd = 0;
+	for (std::vector<UDPSocket *>::const_iterator
```

Comment:
Functionality-wise, no.  But everywhere vectors are used (aside from the nerzhul list -> vector changes) they are treated as arrays.